tools/proofers/dp_proof.js

```diff
@@ -202,6 +202,8 @@ function insertTags(tagOpen, tagClose, replace, retainSelection) {
         if (selection === '') {
             tagOpen = tagOpen.replace(': ', '');
         }
+    } else if (tagOpen === '[** ' && selection) {
+        tagOpen = selection + tagOpen;
```

Comment:
Stupid question, since I can't see it in operation ... does the assignment order matter? As shown below in line 209, the actual order should be tagOpen + selection + tagClose.

Reply:
The tagClose gets added later. I'm just here modifying the tagOpen to be [** selection. TagClose of ] gets added below.

Reply:
That was my assumption -- what confused me was the order of `tagOpen = selection + tagOpen;` in line 206 being `selection + tagOpen` instead of `tagOpen + selection`. (Very literal-minded, here, out of ignorance.)

Reply:
Let's say selection was "Hello". on 206, we convert tagOpen from "[** " to "Hellow[** ". Then later when we update the text area, we'll change "Hello" to tagOpen + selection + tagClose which turns into "Hello[** Hello]". I'll update sandbox with the code once I get off later and can log into and run the update script.

Reply:
Okay, now I understand -- thanks!